### PR TITLE
Add learning objectives, key points, and drop-downs

### PR DESCRIPTION
**Pull request by mahesh-panchal**
- Adds learning objectives
- Adds Key points
- Adds Scope questions
- Adds drop-down sections for parts that should be hidden.

### CODE REVIEW

**Comment by mahesh-panchal** [comment]
Shoot, this is the wrong branch ...

**Comment by mahesh-panchal** [comment]
I'll just rebase when Fran's stuff is merged.

**Comment by FranBonath** [comment]
@mahesh-panchal my stuff is merged now :)

File: src/content/docs/contributing/nf_core_basic_training.md [hunk 1]
```diff
@@ -30,64 +45,72 @@ The course is going to build an (totally unscientific and useless) RNA seq pipel
...
+  a) Adding a nf-core module to your pipeline
 
-**5. Building a nf-core pipeline using the template**
+  b) Adding a local custom module to your pipeline
 
-a) Adding a nf-core module to your pipeline
+  c) Working with Nextflow schema
 
-b) Adding a local custom module to your pipeline
+  d) Linting your modules
```
Comment (FranBonath) [inline src/content/docs/contributing/nf_core_basic_training.md:76]: somehow the indentation for the sub-points is messed up, which makes this section very hard to read.

File: src/content/docs/contributing/nf_core_basic_training.md [hunk 2]
```diff
@@ -30,64 +45,72 @@ The course is going to build an (totally unscientific and useless) RNA seq pipel
...
 </p>
 
-For more information about gitpod, including how to make your own gitpod environement, see the gitpod bytesize talk on youtube (link to the bytesize talk)
+For more information about Gitpod, including how to make your own Gitpod environement, see the Gitpod bytesize talk on youtube (link to the bytesize talk),
+check the [nf-core Gitpod documentation](gitpod/index) or [Gitpod's own documentation](https://www.gitpod.io/docs).
 
-## Explore your Gitpod interface
+<details>
+<summary> Expand this section for instructions to explore your Gitpod environment</summary>
+
```
Comment (FranBonath) [inline src/content/docs/contributing/nf_core_basic_training.md:112]: would be good to have some space between this expandable area and the next header. Currently it looks a bit cramped.

**Review by FranBonath — APPROVED** [review]
Some small things to change, but otherwise LGTM. We can make the changes also in a later round of proof-reading.

**Comment by mahesh-panchal** [comment]
All sick w. Covid. Will fix later.

File: src/content/docs/contributing/nf_core_basic_training.md (hunk 2 above)
Comment (mahesh-panchal) [reply src/content/docs/contributing/nf_core_basic_training.md:112]: We should add a margin then to the style sheet. @mashehu which is the correct file to edit if I want to add a margin to the bottom of the details tag?

**Pull request merged**